fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -230,6 +234,7 @@ def __init__(
         clear_autocast_cache: bool = False,
         force_input_to_fp32: bool = False,
         verbose: bool = False,
+        cpu_offload: bool = False,
```

Comment:
perhaps add a comment line above to separate, like:
```suggestion
        # deprecated args below.
        cpu_offload: bool = False,
```

Reply:
Same comment as above.